server/internal/gst/gst.go

```diff
@@ -131,7 +131,11 @@ func CreateAppPipeline(codecName string, pipelineDevice string, pipelineSrc stri
 			return nil, err
 		}
 
-		pipelineStr = fmt.Sprintf(videoSrc+"video/x-raw,format=I420 ! x264enc threads=4 bitrate=%d byte-stream=true tune=zerolatency speed-preset=veryfast ! video/x-h264,stream-format=byte-stream"+pipelineStr, pipelineDevice, fps, bitrate)
+		vbvbuf := uint(1000)
+		if bitrate > 1000 {
+			vbvbuf = bitrate
+		}
+		pipelineStr = fmt.Sprintf(videoSrc+"video/x-raw,format=NV12 ! x264enc threads=4 bitrate=%d key-int-max=60 vbv-buf-capacity=%d byte-stream=true tune=zerolatency speed-preset=veryfast ! video/x-h264,stream-format=byte-stream"+pipelineStr, pipelineDevice, fps, bitrate, vbvbuf)
```

Comment:
Is there any reason for switching to `format=NV12` ?

Reply:
I found some sources, that NV12 is a preferred format for live streaming (VideoLAN commit ( https://mailman.videolan.org/pipermail/x264-devel/2010-July/007551.html ) and OBS ( https://github.com/obsproject/obs-studio/issues/3517#issuecomment-701115779 )

Other than that no reason. it since it was not the root of the problem, I can switch it back if you want.

Reply:
No, that's ok. I was just curious.